modules/popup/react/README.md

```diff
@@ -398,11 +398,28 @@ useCloseOnOutsideClick(stackRef: React.RefObject<HTMLElement>, onClose: () => vo
 
 Registers global listener for all clicks. It will only call the `onClose` callback if the click
 happened outside the `stackRef` element and its children _and_ the provided `stackRef` element is
-the topmost in the stack. The `stackRef` should be the same as the one passed to `usePopupStack` or
-the `Popper` component since `Popper` uses `usePopupStack` internally.
+the topmost element with this behavior applied in the stack. Adds a `data-behavior-click-outside`
+attribute to track usage of this behavior hook.
 
-This should be used stacked UI elements that are dismissible like Tooltips, Modals, non-modal
-dialogs, dropdown menus, etc.
+The `stackRef` should be the same as the one passed to `usePopupStack` or the `Popper` component
+since `Popper` uses `usePopupStack` internally.
+
+This should be used stacked UI elements that are dismissible like Modals, non-modal dialogs,
```

Comment:
"This should be used to stack UI elements"

"This is used to stack UI elements"?

Reply:
I think I meant "This should be used with stacked UI elements".